src/test/java/net/minecraftforge/debug/client/CustomizeGuiOverlayEventTest.java

```diff
@@ -0,0 +1,35 @@
+package net.minecraftforge.debug.client;
+
+import net.minecraftforge.api.distmarker.Dist;
+import net.minecraftforge.client.event.CustomizeGuiOverlayEvent;
+import net.minecraftforge.eventbus.api.EventPriority;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.common.Mod;
+
+@Mod(CustomizeGuiOverlayEventTest.MODID)
+public class CustomizeGuiOverlayEventTest
+{
+    private static final boolean ENABLED = false;
+
+    public static final String MODID = "custom_gui_event_test";
+    @Mod.EventBusSubscriber(value = Dist.CLIENT, modid = MODID, bus = Mod.EventBusSubscriber.Bus.FORGE)
+    public static class ClientEvents {
+        @SubscribeEvent(priority = EventPriority.NORMAL)
+        public static void onRenderOverlayDebug(CustomizeGuiOverlayEvent.DebugText event) {
+            if(ENABLED) {
+                if(CustomizeGuiOverlayEvent.DebugText.Side.Left.equals(event.getSide())) {
```

Comment:
it's a enum.. you use == not .equals

Reply:
either works fine in most cases but nbd I can change it. .equals just wont throw a compile error if you put in some other object in the .equals() where == would.

Reply:
Ya, but == looks better IMO, for enums it should actually end up being the same thing, but I like the look and compile time double checking. Plus you had to change other things so might as well.